NuGet.config

```diff
@@ -12,20 +12,6 @@
   </activePackageSource>
   <packageSources>
     <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet10" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet10/nuget/v3/index.json" />
-    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <add key="slngen-aggregate" value="https://pkgs.dev.azure.com/devdiv/DevDiv/_packaging/slngen-aggregate/nuget/v3/index.json" />
```

Comment:
Is this feed available to the general public?  We want outside contributors to be able to build the repo.

Reply:
I'm restricted on visibility when creating a nuget feed under DevDiv, but still looking into how this can be made available to the public.

Reply:
Update - I am leaning towards the existing change and am waiting on access to queue a build to mirror nuget packages with dotnet-public. Another alternative is to create a public project and feed which relies on dotnet-tools, dotnet10, and nuget.org as upstream sources. This should be decided by EOD tomorrow, thanks for your patience!